Assets/New Terrain 2.asset.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Ditto above... though are you really using two terrains?

Reply:
two scenes, one was the original neighbourhood. Hoping to be able to reuse it as a background for the school